ccm_web/server/src/lti/lti.service.ts

```diff
@@ -43,17 +44,15 @@ export class LTIService implements BeforeApplicationShutdown {
       { plugin: dbPlugin },
       {
         // Reserved routes
-        appRoute: '/lti',
-        loginRoute: '/lti/login',
-        keysetRoute: '/lti/keys',
+        appRoute: '/',
+        loginRoute: '/login',
+        keysetRoute: '/keys',
```

Comment:
Do these changes mean the URLs in the LTI config used for the LMS will need to change?  Or are these new routes relative to the `/lti` route?

Reply:
Nothing should need to change in the configuration. As you said, these should be relative to `/lti/`. You can see where I have applied this as middleware to that route root in `lti.module.ts`. (It is working, but maybe there's a more correct way to specify this in `forRoutes`; I'll look into it.)

Reply:
If it's working, you don't need to change it on my account.  I just needed a little clarification to be sure I understood how it was working.  If anything is needed, just adding a comment to say these paths are relative to `/lti` would be sufficient.